<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>10</y>
      <width>781</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGraphicsView" name="graphicsView_camera"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>781</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>659</width>
         <height>0</height>
        </size>
       </property>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>601</width>
          <height>31</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Width (pixcel)</string>
         </property>
        </widget>
        <widget class="QSlider" name="horizontalSlider_width">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>10</y>
           <width>311</width>
           <height>16</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="width_label">
         <property name="geometry">
          <rect>
           <x>540</x>
           <y>-1</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Wv</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_width">
         <property name="geometry">
          <rect>
           <x>430</x>
           <y>0</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="frame_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>601</width>
          <height>31</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Hight (pixcel)</string>
         </property>
        </widget>
        <widget class="QSlider" name="horizontalSlider_Length">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>10</y>
           <width>311</width>
           <height>16</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="length_label">
         <property name="geometry">
          <rect>
           <x>540</x>
           <y>0</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Lv</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_length">
         <property name="geometry">
          <rect>
           <x>430</x>
           <y>0</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="frame_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>80</y>
          <width>601</width>
          <height>31</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Exposure Time (us)</string>
         </property>
        </widget>
        <widget class="QSlider" name="horizontalSlider_ExpTime">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>10</y>
           <width>311</width>
           <height>16</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="ExpTime_label">
         <property name="geometry">
          <rect>
           <x>540</x>
           <y>0</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Expv</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_ExpTime">
         <property name="geometry">
          <rect>
           <x>430</x>
           <y>0</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="frame_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>120</y>
          <width>601</width>
          <height>31</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Gain (Raw)</string>
         </property>
        </widget>
        <widget class="QSlider" name="horizontalSlider_Gain_Raw">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>10</y>
           <width>311</width>
           <height>16</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="Gain_Raw_label">
         <property name="geometry">
          <rect>
           <x>540</x>
           <y>0</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Gv</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_gain_raw">
         <property name="geometry">
          <rect>
           <x>430</x>
           <y>0</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="frame_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>160</y>
          <width>601</width>
          <height>31</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Gain (db)</string>
         </property>
        </widget>
        <widget class="QSlider" name="horizontalSlider_Gain_DB">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>10</y>
           <width>311</width>
           <height>16</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="Gain_db_label">
         <property name="geometry">
          <rect>
           <x>540</x>
           <y>0</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Gdbv</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_gain_db">
         <property name="geometry">
          <rect>
           <x>430</x>
           <y>0</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
